lib/plugins/physics.js

```diff
@@ -196,26 +195,37 @@ function inject (bot) {
     })
   }
 
+  let lookingTask = createDoneTask()
+
+  bot.on('move', () => {
+    if (Math.abs(deltaYaw(bot.entity.yaw, lastSentYaw)) < 0.001 && !lookingTask.done) {
```

Comment:
better put the lookingTask.done first in the condition so it doesn't do the computation if not needed

Reply:
You're right... short-circuiting for the win